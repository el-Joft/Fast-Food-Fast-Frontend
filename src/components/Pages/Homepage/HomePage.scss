

.banner{
  background: url('/src/public/images/banner-1.jpg') no-repeat;
  background-size:cover;
  height:450px;
  width:100vw;
  margin-top: -40px;
}

.banner-text{
  margin: 40px;
  width: 40%;
  padding: 60px;
}


.banner-text h2{
  line-height: 3em;
  font-size: 40px;
  color:#42ce9f;
}
.banner-text p{
  font-size: 1.1em;
  color:#42ce9f;
}
.menu-banner{
  background: url('/src/public/images/banner-5.jpg') no-repeat;
  background-size: cover;
  height: 300px;
  width:100vw;
}
.home-banner{
  color: #495057!important;
}

.today-special{
  background: #42ce9f;
  color: white;
  padding: 18px;

}
.today-special p{
  margin-left: 40px;
}
.today-special-food{
  font-family: Handlee;
  font-size: 1.4em;
}
/* MENU */
.menu{
  margin-top: 20px;
}


.sweets-menu{
  background: url('/src/public/images/banner-5.jpg') no-repeat;
  background-size: 100% 100%;
  background-attachment: fixed;
  color: #fff;
  padding: 20px;
}
.sweets-menu h3{
  color: #fff!important;
}

.menu-text h3{
  font-size: 2.5em;
  border: 2px solid #42ce9f;
  padding: 10px;
  text-align: center;
  width: 30%;
  color: #495057;


}
.menu-text a{
  color: #495057;
}


.menu-text p{
  width: 50%;
  text-align: center;
  font-style: italic;
  padding: 10px;
  font-size: 19px;
}
.menu-body{
  display: flex;
  flex-wrap: wrap;
  margin-bottom: 40px;

}

.menu-section {
  display: flex;
  padding: 12px;
  border-bottom: 2px solid #42ce9f;
  width: 40%;
  margin-left: 30px;
}


.menu-price p{
  margin-top: -3px;
  padding: 7px;

}
.menu-food{
  width: 60%;

}
.menu-food h3{
  margin-bottom: 14px;
}
.menu-food p{
  margin-right: 10px;
}
.menu-order{
  margin-top: 10px;
  border: 2px solid #42ce9f;
  background: #42ce9f;
  color: #fff;
  padding: 3px;
  cursor: pointer;
}
.menu-order:hover {
  background: #fff;
  color: #42ce9f;
  border: 1px solid #42ce9f;
}
.full-menu{
  font-size: 25px!important;
  padding: 5px!important;
  width: 20%!important;
  margin-top: 20px;
}




/* ABOUT */
.about-banner{

  background: url('/src/public/images/banner-3.jpg') no-repeat;
  background-size: cover;
  height: 300px;
  width:100vw;

}
.about{
  width: 80%;
  text-align: justify;

}
.info{
  height: 300px;
  width: 100%;
  /* background: #d9d9d9; */
  margin-top: 20px;
  background: url('/src/public/images/banner-4.jpg')no-repeat;
  background-size:cover;
}
.info-text h3{
  font-size: 2.3em;
  font-family: Righteous;
  text-align: center;
  padding: 20px;
}
.info-text h1{
  font-family: Kreon;
  font-size: 3.0em;
  text-align: center;
  padding: 10px;
  color: #ff0a10;
}
.about-paragraph{
  text-align: center;
  padding: 10px;
  font-size: 1.2em;
  line-height: 40px;
  width: 80%;
}

@media (min-width: 320px) and (max-width: 768px) {
  .logo img {
    height: 70px;
    width: 100px;
    float: left;
  }
  .logo-text {
    font-size: 1.2em;
    margin-top: 20px;
  }
  .working-hours, .social, .nav-bar {
    display: none;
  }
  .banner {
    width: 100%;
  }
  .banner-text {
    padding: 7px;
    width: unset;
  }
  .today-special-food {
    font-family: Handlee;
    font-size: 1.4em;
    font-family: Handlee;
    font-size: 1.0em;
  }
  .menu-banner {
    height: 150px;
  }
  .menu-text {
    h3 {
      font-size: 1.2em;
      padding: 1px;
    }
    p {
      font-size: 12px;
    }
  }
  .menu-section {
    margin: 10px;
    width: 100%;
  }
  .full-menu {
    font-size: 15px !important;
    padding: 10px !important;
    width: 30% !important;
  }
  .about-paragraph {
    text-align: justify;
    padding: 9px;
    font-size: 1.0em;
    line-height: 30px;
    width: 100%;
  }
  .info-text h3 {
    font-size: 1.7em;
  }
  .banner-text h2 {
    font-size: 30px;
  }
  .checkout {
    width: 90%;
  }
  .top-button {
    width: 70%;
  }
  .history-head-text {
    font-size: 1.0em;
  }
  .success-text {
    margin-bottom: 30px;
  }
  .successful {
    height: 340px;
  }
}
