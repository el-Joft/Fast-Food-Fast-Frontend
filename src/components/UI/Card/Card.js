/* eslint-disable react/destructuring-assignment */
import React, { Component } from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import { getAllMenus } from '../../../actions/getMenusAction';
import { addCart } from '../../../actions/orderAction';


class Card extends Component {
  componentDidMount() {
    this.props.getAllMenus();
  }

  addToCart(menuData) {
    const item = Object.assign({}, menuData, { quantity: 1 });
    this.props.addCart(item);
  }

  renderCard = () => {
    const { menu } = this.props;
    if (menu.menu !== null) {
      return menu.menu.map(item => (
        <div key={item.id} className='menu-section'>
          <div className="menu-food">
            <h3>{item.name}</h3>
            <p>{item.description}</p>
          </div>
          <div className="menu-price">
            <p>
              <strike>N</strike>
              {item.price}
            </p>
            <div to="/checkout">
              {' '}
              <span
                role="searchbox"
                onKeyPress={() => this.addToCart(item)}
                onClick={menuData => this.addToCart(item)}
                tabIndex="-1"
                className="menu-order"
              >
                Make Order
              </span>
            </div>
          </div>

        </div>
      ));
    }
  }

  render() {
    return (
      <div className="menu-body">
        {this.renderCard()}
      </div>
    );
  }
}

Card.propTypes = {
  getAllMenus: PropTypes.func.isRequired,
  addCart: PropTypes.func.isRequired,
  menu: PropTypes.object.isRequired,
  errors: PropTypes.object.isRequired,
};

// we get values here using the this.props.errors/auth

const mapStateToProps = state => ({
  menu: state.menu,
  errors: state.errors,
  order: state.order,
});

export default connect(mapStateToProps, { getAllMenus, addCart })(Card);
