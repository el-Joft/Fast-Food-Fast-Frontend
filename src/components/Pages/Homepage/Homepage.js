/* eslint-disable import/no-named-as-default */
/* eslint-disable react/forbid-prop-types */
/* eslint-disable react/no-unused-prop-types */
/* eslint-disable react/no-unescaped-entities */
import React, { Component } from 'react';
import './HomePage.scss';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import Card from '../../UI/Card/Card';
// import { getOrders } from '../../../actions/orderAction';

export class Homepage extends Component {
  state = {

  }

  render() {
    return (
      <div>
        <section className="container">
          <div className="banner" id="banner">
            <div>
              <div className="banner-text">
                <h2 className="home-banner">Fast Food Fast</h2>
                <p className="home-banner">
Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks,
                            and desserts at affordable price.
                            We are glad to offer you the best food in the area to everyone.

                </p>
              </div>

            </div>
          </div>
        </section>
        <div className="today-special">
          <p>
              TODAY'S SPECIAL :
            <span className="today-special-food"> Amala with Ewedu plus Goat Meat</span>
          </p>
        </div>
        <div className="menu">
          <div className="menu-text">
            <h3>Our Menu</h3>
            <p> "Our talented chefs prepare best meals for you"</p>
          </div>
          <Card />
          <div className="menu-text">
            <h3 className="full-menu"><Link to='/menu'>View Full Menu</Link></h3>
          </div>
        </div>
        <section className="info">
          <div className="info-text">
            <h3>FAST FOOD FAST</h3>
          </div>
          <div className="about-paragraph">
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit.
              Nulla, voluptates laborum blanditiis vitae
              recusandae eaque ea ipsa magnam.
              Perspiciatis labore veniam recusandae excepturi
              repellendus ea molestias est reprehenderit veritatis sequi.
            </p>

          </div>
        </section>
      </div>
    );
  }
}

Homepage.propTypes = {
  auth: PropTypes.object.isRequired,
  isAuthenticated: PropTypes.bool.isRequired,
  // history: PropTypes.func.isRequired,
};

const mapStateToProps = state => ({
  auth: state.auth,
});

export default connect(mapStateToProps, {})(Homepage);
