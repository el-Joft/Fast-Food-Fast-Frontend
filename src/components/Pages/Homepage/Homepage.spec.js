/* eslint-disable react/react-in-jsx-scope */
import { shallow } from 'enzyme';
import { Homepage } from './Homepage';
import { Card } from '../../UI/Card/Card';

describe('rendering', () => {
  let wrapper;
  it('should render the component', () => {
    wrapper = shallow(<Homepage />);
    expect(wrapper.length).toBe(1);
  });

  it('should render the component', () => {
    wrapper = shallow(<Homepage />);
    expect(wrapper.find(Card)).toEqual({});
  });

  it('should render the component', () => {
    wrapper = shallow(<Homepage />);
    expect(wrapper).toMatchSnapshot();
  });
});
