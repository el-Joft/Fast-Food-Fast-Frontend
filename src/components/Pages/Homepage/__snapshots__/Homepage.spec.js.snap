// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render the component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Homepage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section
          className="container"
        >
          <div
            className="banner"
            id="banner"
          >
            <div>
              <div
                className="banner-text"
              >
                <h2
                  className="home-banner"
                >
                  Fast Food Fast
                </h2>
                <p
                  className="home-banner"
                >
                  Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                </p>
              </div>
            </div>
          </div>
        </section>,
        <div
          className="today-special"
        >
          <p>
            TODAY'S SPECIAL :
            <span
              className="today-special-food"
            >
               Amala with Ewedu plus Goat Meat
            </span>
          </p>
        </div>,
        <div
          className="menu"
        >
          <div
            className="menu-text"
          >
            <h3>
              Our Menu
            </h3>
            <p>
               "Our talented chefs prepare best meals for you"
            </p>
          </div>
          <Connect(Card) />
          <div
            className="menu-text"
          >
            <h3
              className="full-menu"
            >
              <Link
                replace={false}
                to="/menu"
              >
                View Full Menu
              </Link>
            </h3>
          </div>
        </div>,
        <section
          className="info"
        >
          <div
            className="info-text"
          >
            <h3>
              FAST FOOD FAST
            </h3>
          </div>
          <div
            className="about-paragraph"
          >
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
            </p>
          </div>
        </section>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="banner"
            id="banner"
          >
            <div>
              <div
                className="banner-text"
              >
                <h2
                  className="home-banner"
                >
                  Fast Food Fast
                </h2>
                <p
                  className="home-banner"
                >
                  Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                </p>
              </div>
            </div>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div>
              <div
                className="banner-text"
              >
                <h2
                  className="home-banner"
                >
                  Fast Food Fast
                </h2>
                <p
                  className="home-banner"
                >
                  Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                </p>
              </div>
            </div>,
            "className": "banner",
            "id": "banner",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="banner-text"
              >
                <h2
                  className="home-banner"
                >
                  Fast Food Fast
                </h2>
                <p
                  className="home-banner"
                >
                  Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                </p>
              </div>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="home-banner"
                  >
                    Fast Food Fast
                  </h2>,
                  <p
                    className="home-banner"
                  >
                    Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                  </p>,
                ],
                "className": "banner-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Fast Food Fast",
                    "className": "home-banner",
                  },
                  "ref": null,
                  "rendered": "Fast Food Fast",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.",
                    "className": "home-banner",
                  },
                  "ref": null,
                  "rendered": "Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            TODAY'S SPECIAL :
            <span
              className="today-special-food"
            >
               Amala with Ewedu plus Goat Meat
            </span>
          </p>,
          "className": "today-special",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "TODAY'S SPECIAL :",
              <span
                className="today-special-food"
              >
                 Amala with Ewedu plus Goat Meat
              </span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "TODAY'S SPECIAL :",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " Amala with Ewedu plus Goat Meat",
                "className": "today-special-food",
              },
              "ref": null,
              "rendered": " Amala with Ewedu plus Goat Meat",
              "type": "span",
            },
          ],
          "type": "p",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="menu-text"
            >
              <h3>
                Our Menu
              </h3>
              <p>
                 "Our talented chefs prepare best meals for you"
              </p>
            </div>,
            <Connect(Card) />,
            <div
              className="menu-text"
            >
              <h3
                className="full-menu"
              >
                <Link
                  replace={false}
                  to="/menu"
                >
                  View Full Menu
                </Link>
              </h3>
            </div>,
          ],
          "className": "menu",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Our Menu
                </h3>,
                <p>
                   "Our talented chefs prepare best meals for you"
                </p>,
              ],
              "className": "menu-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Our Menu",
                },
                "ref": null,
                "rendered": "Our Menu",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " \\"Our talented chefs prepare best meals for you\\"",
                },
                "ref": null,
                "rendered": " \\"Our talented chefs prepare best meals for you\\"",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3
                className="full-menu"
              >
                <Link
                  replace={false}
                  to="/menu"
                >
                  View Full Menu
                </Link>
              </h3>,
              "className": "menu-text",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/menu"
                >
                  View Full Menu
                </Link>,
                "className": "full-menu",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "View Full Menu",
                  "replace": false,
                  "to": "/menu",
                },
                "ref": null,
                "rendered": "View Full Menu",
                "type": [Function],
              },
              "type": "h3",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="info-text"
            >
              <h3>
                FAST FOOD FAST
              </h3>
            </div>,
            <div
              className="about-paragraph"
            >
              <p>
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
              </p>
            </div>,
          ],
          "className": "info",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3>
                FAST FOOD FAST
              </h3>,
              "className": "info-text",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "FAST FOOD FAST",
              },
              "ref": null,
              "rendered": "FAST FOOD FAST",
              "type": "h3",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p>
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
              </p>,
              "className": "about-paragraph",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.",
              },
              "ref": null,
              "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.",
              "type": "p",
            },
            "type": "div",
          },
        ],
        "type": "section",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="container"
          >
            <div
              className="banner"
              id="banner"
            >
              <div>
                <div
                  className="banner-text"
                >
                  <h2
                    className="home-banner"
                  >
                    Fast Food Fast
                  </h2>
                  <p
                    className="home-banner"
                  >
                    Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                  </p>
                </div>
              </div>
            </div>
          </section>,
          <div
            className="today-special"
          >
            <p>
              TODAY'S SPECIAL :
              <span
                className="today-special-food"
              >
                 Amala with Ewedu plus Goat Meat
              </span>
            </p>
          </div>,
          <div
            className="menu"
          >
            <div
              className="menu-text"
            >
              <h3>
                Our Menu
              </h3>
              <p>
                 "Our talented chefs prepare best meals for you"
              </p>
            </div>
            <Connect(Card) />
            <div
              className="menu-text"
            >
              <h3
                className="full-menu"
              >
                <Link
                  replace={false}
                  to="/menu"
                >
                  View Full Menu
                </Link>
              </h3>
            </div>
          </div>,
          <section
            className="info"
          >
            <div
              className="info-text"
            >
              <h3>
                FAST FOOD FAST
              </h3>
            </div>
            <div
              className="about-paragraph"
            >
              <p>
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
              </p>
            </div>
          </section>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="banner"
              id="banner"
            >
              <div>
                <div
                  className="banner-text"
                >
                  <h2
                    className="home-banner"
                  >
                    Fast Food Fast
                  </h2>
                  <p
                    className="home-banner"
                  >
                    Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                  </p>
                </div>
              </div>
            </div>,
            "className": "container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div>
                <div
                  className="banner-text"
                >
                  <h2
                    className="home-banner"
                  >
                    Fast Food Fast
                  </h2>
                  <p
                    className="home-banner"
                  >
                    Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                  </p>
                </div>
              </div>,
              "className": "banner",
              "id": "banner",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="banner-text"
                >
                  <h2
                    className="home-banner"
                  >
                    Fast Food Fast
                  </h2>
                  <p
                    className="home-banner"
                  >
                    Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                  </p>
                </div>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="home-banner"
                    >
                      Fast Food Fast
                    </h2>,
                    <p
                      className="home-banner"
                    >
                      Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.
                    </p>,
                  ],
                  "className": "banner-text",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Fast Food Fast",
                      "className": "home-banner",
                    },
                    "ref": null,
                    "rendered": "Fast Food Fast",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.",
                      "className": "home-banner",
                    },
                    "ref": null,
                    "rendered": "Welcome to Food Delivery, a place where you can order your favorite restaurant dishes, drinks, and desserts at affordable price. We are glad to offer you the best food in the area to everyone.",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            "type": "div",
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              TODAY'S SPECIAL :
              <span
                className="today-special-food"
              >
                 Amala with Ewedu plus Goat Meat
              </span>
            </p>,
            "className": "today-special",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "TODAY'S SPECIAL :",
                <span
                  className="today-special-food"
                >
                   Amala with Ewedu plus Goat Meat
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "TODAY'S SPECIAL :",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Amala with Ewedu plus Goat Meat",
                  "className": "today-special-food",
                },
                "ref": null,
                "rendered": " Amala with Ewedu plus Goat Meat",
                "type": "span",
              },
            ],
            "type": "p",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="menu-text"
              >
                <h3>
                  Our Menu
                </h3>
                <p>
                   "Our talented chefs prepare best meals for you"
                </p>
              </div>,
              <Connect(Card) />,
              <div
                className="menu-text"
              >
                <h3
                  className="full-menu"
                >
                  <Link
                    replace={false}
                    to="/menu"
                  >
                    View Full Menu
                  </Link>
                </h3>
              </div>,
            ],
            "className": "menu",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3>
                    Our Menu
                  </h3>,
                  <p>
                     "Our talented chefs prepare best meals for you"
                  </p>,
                ],
                "className": "menu-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Our Menu",
                  },
                  "ref": null,
                  "rendered": "Our Menu",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " \\"Our talented chefs prepare best meals for you\\"",
                  },
                  "ref": null,
                  "rendered": " \\"Our talented chefs prepare best meals for you\\"",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3
                  className="full-menu"
                >
                  <Link
                    replace={false}
                    to="/menu"
                  >
                    View Full Menu
                  </Link>
                </h3>,
                "className": "menu-text",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/menu"
                  >
                    View Full Menu
                  </Link>,
                  "className": "full-menu",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "View Full Menu",
                    "replace": false,
                    "to": "/menu",
                  },
                  "ref": null,
                  "rendered": "View Full Menu",
                  "type": [Function],
                },
                "type": "h3",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="info-text"
              >
                <h3>
                  FAST FOOD FAST
                </h3>
              </div>,
              <div
                className="about-paragraph"
              >
                <p>
                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
                </p>
              </div>,
            ],
            "className": "info",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  FAST FOOD FAST
                </h3>,
                "className": "info-text",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "FAST FOOD FAST",
                },
                "ref": null,
                "rendered": "FAST FOOD FAST",
                "type": "h3",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p>
                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.
                </p>,
                "className": "about-paragraph",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.",
                },
                "ref": null,
                "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla, voluptates laborum blanditiis vitae recusandae eaque ea ipsa magnam. Perspiciatis labore veniam recusandae excepturi repellendus ea molestias est reprehenderit veritatis sequi.",
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "section",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
